### PR TITLE
aggregates: new output parsing

### PR DESCRIPTION
**Pull request by sileht**
This aggregates API have changed the output. This change do
the same for the client.

### CODE REVIEW

**Comment by sileht** [comment]
Pull-request updated, HEAD is now 9c7a3e81ff8dc1da6b83844d835f5f1562271814

**Comment by sileht** [comment]
Pull-request updated, HEAD is now 655c0de7650bd5b0fd36803a988596aa15332fc9

File: tox.ini [hunk 1]
```diff
@@ -45,5 +45,4 @@ exclude=.venv,.git,.tox,dist,doc,*lib/python*,*egg,build
 python = 3.6: py36, pep8, docs-gnocchi.xyz
 
 [pytest]
-addopts = --verbose --numprocesses=auto
 norecursedirs = .tox
```
Comment (jd) [inline tox.ini]: looks unrelated?

**Comment by sileht** [comment]
Pull-request updated, HEAD is now 45eb925ec483ed91339422df620ceac4012c1508
